sips/sip-x/sip-x-nft-metadata.md

```diff
@@ -0,0 +1,295 @@
+        "version": {
+            "type": number,
+            "description": "Version of the JSON schema for metadata. For this SIP, the version number must be `1`."
+        },
+        "name": {
+            "type": "string",
+            "description": "Identifies the asset which this token represents"
+        },
+        "description": {
+            "type": "string",
+            "description": "Describes the asset which this token represents"
+        },
+        "image": {
+            "type": "string",
+            "description": "A URI pointing to a resource with mime type image/* representing the asset to which this token represents. Consider making any images at a width between 320 and 1080 pixels and aspect ratio between 1.91:1 and 4:5 inclusive."
+        },
+        "attributes": {
+            "type": "array",
+            "description": "Arbitrary attributes. Values may be strings, numbers, object or arrays."
+            "items: {
+                "type": "object",
+                "required": ["value"],
+                "properties": {
+                    "display_type": "string",
+                    "trait_type": "string",
+                    "value": {"oneOf": [{"type": "object"}, {"type": "string"}, {"type": "number"}, {"type: "array"}},
+                }
+            }
+        },
+        "properties": {
+            "type": "object",
+            "description": "Arbitrary properties. Values may be strings, numbers, object or arrays."
+        },
+        "localization": {
+            "type": "object",
+            "required": ["uri", "default", "locales"],
+            "properties": {
+                "uri": {
+                    "type": "string",
+                    "description": "The URI pattern to fetch localized data from. This URI should contain the substring `{locale}` which will be replaced with the appropriate locale value before sending the request. See section about localization for more rules"
+                },
+                "default": {
+                    "type": "string",
+                    "description": "The locale of the default data within the base JSON"
+                },
+                "locales": {
+                    "type": "array",
+                    "description": "The list of locales for which data is available. These locales should conform to those defined in the Unicode Common Locale Data Repository (http://cldr.unicode.org/)."
+                }
+            }
+        }
+    }
+}
+```
+
+The lengths of string values is not restricted. Nowadays, clients should be smart enough to deal with values of different lengths.
+
+### Example
+token101.json
+```
+{
+  "version": 1,
+  "name": "Foo #101",
+  "image": "ipfs://somerandomecid",
+  "attributes": [
+     { 
+      "trait_type": "hair",      
+      "value": "red",
+    },
+    {
+      "trait_type": "strength",
+      "display_type": "number",
+      "value": 99,
+    },    
+  ],
+  "properties": {
+    { 
```

Comment:
is this supposed to be nested or instead:
```
"properties": {
    "collection":  "Foo Collection",
    "collection_size":  "10000" 
}
```

Reply:
You are correct. Addressed in 661ba2a073b48aa4a45e30251182adff9a2f4057